Content.Server/Sticky/Systems/StickySystem.cs

```diff
@@ -25,12 +28,57 @@ public override void Initialize()
         base.Initialize();
         SubscribeLocalEvent<StickSuccessfulEvent>(OnStickSuccessful);
         SubscribeLocalEvent<UnstickSuccessfulEvent>(OnUnstickSuccessful);
+
         SubscribeLocalEvent<StickyComponent, AfterInteractEvent>(OnAfterInteract);
         SubscribeLocalEvent<StickyComponent, GetVerbsEvent<Verb>>(AddUnstickVerb);
+
+        SubscribeLocalEvent<InsertMaterialAttemptEvent>(OnInsertMaterialAttemptEvent);
+        SubscribeLocalEvent<StackSplitAttemptEvent>(OnStackSplitAttemptEvent);
+        SubscribeLocalEvent<StackMergeAttemptEvent>(OnStackMergeAttemptEvent);
```

Comment:
these need to be directed to components. presumably `HasEntityStuckOn`

Reply:
if the issue is that you need to check the used and the target, then the event should just be raised on both.

Reply:
What ? I don't really understand what you mean. The difficulty is that I need to check wether or not the entity is stuck on something AND if it has something stuck to it. The improvement I plan to make is generalize all of it into one simple event that checks.